execute_process(COMMAND ${TEST} RESULT_VAR HAD_ERROR)
